### DISCUSSION TITLE
Need to find angle between two long,lat points (drawn line between them) to show label on top of it

### DISCUSSION THREAD

**Discussion by vissharm** [visgl/deck.gl#6165] [Q&A] [answered]
https://github.com/visgl/deck.gl/issues/6162

Please find the calculations in above link.  These are still not aligning the text parallel to link. Any help ? Got stuck.

**Comment by vissharm** [comment]
turf bearing angle is also aligning label somewhere else.... my calculations are better than that.

**Comment by Pessimistress** [comment, marked as answer]
```js
    let angle = 90 - turf.rhumbBearing(from, to);
    if (Math.abs(angle) > 90) {
      angle += 180;
    }
```

**Reply by vissharm** [reply]
Thanks @Pessimistress

**Reply by vissharm** [reply]
if (Math.abs(angle) > 90) {
      angle += 180;
    }
    
    This helped.. I was doing something mess around here....

**Reply by vissharm** [reply]
> ```js
>     let angle = 90 - turf.rhumbBearing(from, to);
>     if (Math.abs(angle) > 90) {
>       angle += 180;
>     }
> ```

@Pessimistress in some cases abgle is coming opposie to the slope of line otherwise its perfectly correct.
